wfe2/wfe.go

```diff
@@ -2273,6 +2292,45 @@ func (wfe *WebFrontEndImpl) NewOrder(
 
 	logEvent.DNSNames = names
 
+	if features.Get().CheckIdentifiersPaused {
+		// Check if any of the identifiers are paused.
+		orderIdentifiers := make([]*sapb.Identifier, 0, len(names))
+		for _, name := range names {
+			ident, exists := identMap[name]
+			if exists {
+				orderIdentifiers = append(orderIdentifiers, ident)
+			}
+		}
+		paused, err := wfe.sa.CheckIdentifiersPaused(ctx, &sapb.PauseRequest{
+			RegistrationID: acct.ID,
+			Identifiers:    orderIdentifiers,
+		})
+		if err != nil {
+			wfe.sendError(response, logEvent, web.ProblemDetailsForError(err, "Error checking if identifiers are paused"), err)
+			return
+		}
+
+		if len(paused.Identifiers) > 0 {
+			// Some or all of the identifiers are paused.
+			pausedValues := make([]string, len(paused.Identifiers))
+			for i, ident := range paused.Identifiers {
+				pausedValues[i] = ident.Value
+			}
```

Comment:
```suggestion
			pausedValues := make([]string, 0, len(paused.Identifiers))
			for _, ident := range paused.Identifiers {
				pausedValues = append(pausedValues, ident.Value)
			}
```

This matches the style used above and is IMO a little more mistake-proof.